catalog/transformer/catalog-transformer-pptx/src/test/java/ddf/catalog/transformer/input/pptx/PptxInputTransformerTest.java

```diff
@@ -0,0 +1,226 @@
+            throws IOException, CatalogTransformerException, InterruptedException {
+
+        try (XMLSlideShow ss = new XMLSlideShow()) {
+            try (ByteArrayOutputStream os = new ByteArrayOutputStream()) {
+                ss.write(os);
+
+                try (ByteArrayInputStream inStr = new ByteArrayInputStream(os.toByteArray())) {
+                    PptxInputTransformer t = new PptxInputTransformer(inputTransformer);
+                    Metacard m = t.transform(inStr);
+                    assertNull(m.getThumbnail());
+                }
+            }
+
+        }
+    }
+
+    @Test
+    public void testWithTitle()
+            throws IOException, CatalogTransformerException, InterruptedException {
+
+        try (XMLSlideShow ss = new XMLSlideShow()) {
+            ss.createSlide();
+            ss.getProperties()
+                    .getCoreProperties()
+                    .setTitle("TheTitle");
+            try (ByteArrayOutputStream os = new ByteArrayOutputStream()) {
+                ss.write(os);
+
+                try (ByteArrayInputStream inStr = new ByteArrayInputStream(os.toByteArray())) {
+                    PptxInputTransformer t = new PptxInputTransformer(inputTransformer);
+                    Metacard m = t.transform(inStr);
+                    assertEquals("TheTitle", m.getTitle());
+                }
+            }
+
+        }
+    }
+
+    @Test
+    public void testCreatedDateWith()
+            throws IOException, CatalogTransformerException, InterruptedException {
+
+        try (XMLSlideShow ss = new XMLSlideShow()) {
+            ss.createSlide();
+            // the date must be rounded to the second because when POI serializes the document to XML, it only uses
+            // one second precision
+            Date d =
+                    new Date(TimeUnit.SECONDS.toMillis(TimeUnit.MILLISECONDS.toSeconds(new Date().getTime())));
+
+            ss.getProperties()
+                    .getCoreProperties()
+                    .setCreated(new Nullable<>(d));
+            try (ByteArrayOutputStream os = new ByteArrayOutputStream()) {
+                ss.write(os);
+
+                try (ByteArrayInputStream inStr = new ByteArrayInputStream(os.toByteArray())) {
+                    PptxInputTransformer t = new PptxInputTransformer(inputTransformer);
+                    Metacard m = t.transform(inStr);
+                    assertEquals(d.getTime(),
+                            m.getCreatedDate()
+                                    .getTime());
+                }
+            }
+
+        }
+    }
+
+    @Test
+    public void testModifiedDateWith()
+            throws IOException, CatalogTransformerException, InterruptedException {
+
+        try (XMLSlideShow ss = new XMLSlideShow()) {
+            ss.createSlide();
+            // the date must be rounded to the second because when POI serializes the document to XML, it only uses
+            // one second precision
```

Comment:
Remove comment. Since you're using it twice this is the perfect candidate for a private method ... and then you can block this comment over that method. Win-win.